packages/features/bookings/lib/handleNewBooking/createBooking.ts

```diff
@@ -56,13 +57,16 @@ type CreateBookingParams = {
 function updateEventDetails(
   evt: CalendarEvent,
   originalRescheduledBooking: OriginalRescheduledBooking | null,
-  changedOrganizer: boolean
+  changedOrganizer: boolean,
+  isLocationChanged: boolean
```

Comment:
Imo if these are Boolean these shouldnt be variables but functions, but I don't see why they're Boolean in the first place